### ISSUE TITLE
`annotate_sv` duplicates BCL6-MYC rearrangements

### ISSUE THREAD

**Issue by lkhilton** [morinlab/GAMBLR.utils#47]
This command returns a clear BCL6::MYC event: 

```
sv_data <- get_combined_sv(
          these_sample_ids = "CLC03336",
          oncogenes = "MYC")
```

```
  CHROM_A   START_A     END_A CHROM_B   START_B     END_B manta_name       SCORE STRAND_A STRAND_B tumour_sample_id normal_sample_id VAF_tumour    DP gridss_name ANNOTATION_A DIST_TO_ANNOTATION_A ANNOTATION_B
1 3       187477179 187477181 8       128893895 128893897 MantaBND:0:2025…    NA +        -        CLC03336         14-11247N             0.175   383 gridss68fb… BCL6                       -13924 MYC         
2 3       187480498 187480500 8       128893597 128893599 MantaBND:0:2025…    NA -        +        CLC03336         14-11247N             0.315   470 gridss68bf… BCL6                       -17243 MYC  
```

But if I run `annotate_sv` on the above I get: 

```
   chrom1    start1      end1 chrom2    start2      end2   name score strand1 strand2 tumour_sample_id   gene partner   fusion
1:      3 187477179 187477181      8 128893895 128893897      .    NA       +       -         CLC03336   BCL6    <NA>  NA-BCL6
2:      3 187480498 187480500      8 128893597 128893599      .    NA       -       +         CLC03336   BCL6    <NA>  NA-BCL6
3:      3 187477179 187477181      8 128893895 128893897      .    NA       +       -         CLC03336    MYC    BCL6 BCL6-MYC
4:      3 187480498 187480500      8 128893597 128893599      .    NA       -       +         CLC03336    MYC    BCL6 BCL6-MYC
```

This is with the latest GAMBLR.utils just installed today.

**Comment by Kdreval** [comment]
I'll take a look at this

**Comment by Kdreval** [comment]
Got the same error. Will remove the rows where there is NA while a value is present for partner in the same group. Will add fix for this in PR #53
